EIPS/eip-4362.md

```diff
@@ -0,0 +1,270 @@
+---
+eip: 4362
+title: EIP-4361 Capabilities Extension (CapGrok)
```

Comment:
```suggestion
title: Sign-In with Ethereum Capabilities
```
Don't use `:` in titles. It causes inconsistency with jekyll (since it tries to interpret it as a key named `title: Sign-In with Ethereum` and a value of `Capabilities`.

CC @SamWilsn so you know for future reference.